test/integration/controllers/admin.spec.js

```diff
@@ -0,0 +1,64 @@
+const { serverInit, serverCleanup } = require('~/test/setup')
+const {
+  roles: { ADMIN }
+} = require('~/consts/auth')
+const User = require('~/models/user')
+const Role = require('~/models/role')
+const { USER_NOT_REGISTERED } = require('~/consts/errors')
+const { expectError } = require('~/test/helpers')
+
+describe('Admin controller', () => {
+  let app, server
+
+  beforeAll(async () => {
+    ;({ app, server } = await serverInit())
+  })
+
+  afterAll(async () => {
+    await serverCleanup(server)
+  })
+
+  it('getAdmins', async () => {
```

Comment:
Maybe change the naming to `getAdmins endpoint`, and the same for `getAdmin endpoint` below